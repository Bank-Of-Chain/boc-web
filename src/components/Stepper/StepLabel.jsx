import { withStyles } from "@material-ui/core/styles";
import StepLabel from "@material-ui/core/StepLabel";

const BocStepLabel = withStyles({
  label: {
    color: "#fff",
  },
  active: {
    color: "rgba(255, 255, 255, 0.87) !important",
  },
  completed: {
    color: "rgba(255, 255, 255, 0.87) !important",
  },
})(StepLabel);

export default BocStepLabel;
