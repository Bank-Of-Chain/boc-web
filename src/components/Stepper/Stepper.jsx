import { withStyles } from '@material-ui/core/styles'
import Stepper from '@material-ui/core/Stepper'

const BocStepper = withStyles({
  root: {
    background: 'transparent'
  }
})(Stepper)

export default BocStepper
