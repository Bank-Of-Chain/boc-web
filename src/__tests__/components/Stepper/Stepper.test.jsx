import React from 'react'
import { render } from '@testing-library/react'
import Stepper from '@/components/Stepper/Stepper'
import StepConnector from '@/components/Stepper/StepConnector'
import StepIcon from '@/components/Stepper/StepIcon'
import StepLabel from '@/components/Stepper/StepLabel'

import { describe, expect, it } from 'vitest'

describe('xxxxxxxx', () => {
  it('Stepper Component Render', () => {
    const { asFragment } = render(<Stepper />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('StepConnector Component Render', () => {
    const { asFragment } = render(<StepConnector />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('StepIcon Component Render', () => {
    const { asFragment } = render(<StepIcon />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('StepLabel Component Render', () => {
    const { asFragment } = render(<StepLabel />)
    expect(asFragment()).toMatchSnapshot()
  })
})
