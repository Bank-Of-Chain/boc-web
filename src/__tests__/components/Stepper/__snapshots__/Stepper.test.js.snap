// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepConnector Component Render 1`] = `
<DocumentFragment>
  <div
    class="MuiStepConnector-root MuiStepConnector-horizontal"
  >
    <span
      class="MuiStepConnector-line WithStyles(ForwardRef(StepConnector))-line-5 MuiStepConnector-lineHorizontal"
    />
  </div>
</DocumentFragment>
`;

exports[`StepIcon Component Render 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-6"
  >
    <div
      class="makeStyles-circle-8"
    />
  </div>
</DocumentFragment>
`;

exports[`StepLabel Component Render 1`] = `
<DocumentFragment>
  <span
    class="MuiStepLabel-root MuiStepLabel-horizontal"
  >
    <span
      class="MuiStepLabel-labelContainer"
    />
  </span>
</DocumentFragment>
`;

exports[`Stepper Component Render 1`] = `
<DocumentFragment>
  <div
    class="MuiPaper-root MuiStepper-root WithStyles(ForwardRef(Stepper))-root-1 MuiStepper-horizontal MuiPaper-elevation0"
  />
</DocumentFragment>
`;
